src/common/components/footer/Footer.tsx

```diff
@@ -0,0 +1,145 @@
+/** @jsxImportSource @emotion/react */
+import { FC } from "react";
+import tw, { styled } from "twin.macro"; // eslint-disable-line
+import { FooterLogo } from "assets/icons";
+import { Discord, Github, Medium, Twitter } from "assets/icons";
+
+const Footer: FC = () => {
+  return (
+    <StyledFooter>
+      <div tw="flex">
+        <div className="logo-wrapper">
+          <FooterLogo />
+          <div tw="mt-4 opacity-75">Risk Labs &copy; 2021</div>
+        </div>
+        <div className="link-wrapper">
+          <ul className="links">
+            {/* TBD: Need links, asked Tom */}
+            <li className="link">How UMA Works</li>
+            <li className="link">Getting Started</li>
+            <li className="link">Docs</li>
+            <li className="link">Vote</li>
+            <li className="link">FAQs</li>
+            <li className="link">Careers</li>
+            <li className="link">Contact</li>
+          </ul>
+        </div>
+        <div className="subscribe-wrapper">
+          <div className="subscribe">
+            <h3 className="header">Get UMA Updates</h3>
+            <p className="sub-text">
+              Sign up for our newsletter to stay updated about the UMA project.
+            </p>
+            <div className="sm-links-wrapper">
+              {socialLinks.map(({ logo, url }, index) => {
+                return (
+                  <a
+                    key={index}
+                    href={url}
+                    tw="pr-2 py-3 flex-grow"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    {logo}
+                  </a>
+                );
+              })}
+            </div>
+          </div>
+        </div>
+      </div>
+    </StyledFooter>
+  );
+};
+
+const socialLinks = [
+  {
+    url: "https://medium.com/uma-project",
+    logo: <Medium className="sm-logo" />,
+  },
+  {
+    url: "https://github.com/UMAprotocol",
+    logo: <Github className="sm-logo" />,
+  },
+  {
+    url: "https://twitter.com/UMAprotocol",
+    logo: <Twitter className="sm-logo" />,
+  },
+  {
+    url: "https://discord.umaproject.org",
+    logo: <Discord className="sm-logo" />,
+  },
+];
+
+const StyledFooter = styled.div`
```

Comment:
@Gamaranto want to bring your attention to this style of css writing and get your thoughts on it

Reply:
Yeah I was partially experimenting with how I could organize the code. I wasn't fully happy with all of the CSS implementation.

I do think it's probably better to put the tw logic inside the main styled component, as you are likely to add *some* general styling that won't be covered by a tw class.